ad9959/src/lib.rs

```diff
@@ -4,6 +4,22 @@ use bit_field::BitField;
 use bitflags::bitflags;
 use embedded_hal::{blocking::delay::DelayUs, digital::v2::OutputPin};
 
+/// An arbitrary frequency that sits between the 160 MHz to 255 MHz zone, where system clock
+/// frequency is not supported by both high frequency and low frequency VCO settings.
+const VCO_MIN_FREQUENCY: f32 = 200e6;
```

Comment:
We don't allow configuration in the 160-255MHz range based on the clocking checks below. We should remove this constant and just use either the min (high gain) or max (low gain) as a switch for the VCO gain bit.